src/backoffice/shared/components/history-ui-maker/history-ui-maker.element.ts

```diff
@@ -0,0 +1,160 @@
+import { css, html, nothing } from 'lit';
+import { UUITextStyles } from '@umbraco-ui/uui-css/lib';
+import { customElement, property, state } from 'lit/decorators.js';
+import { UmbLitElement } from '@umbraco-cms/element';
+
+export interface logItem {
+	userName: string;
+	timestamp: string;
+	logType: logType;
+	parameters: string;
+}
+
+export type logType = 'Save' | 'Publish' | 'ContentVersionPreventCleanup' | 'ContentVersionEnableCleanup';
+
+@customElement('umb-history-ui-maker')
```

Comment:
I would name this component `umb-history-list`

where each entry is a component on its own. called something like `umb-history-item`.


Looks good :-)

Reply:
I have updated the names. Then we can get this merged.

It looks really good 🥇